configuration/persistence.md

```diff
@@ -139,20 +155,40 @@ You can easily imagine that you can implement very powerful rules using this fea
 
```

Comment:
In the unchanged part above, please capitalize headlines

Reply:
Okay - will ensure all headlines are capitalized in the entire article.

Reply:
This comment has been addressed